### ISSUE TITLE
Add label to attachment icon

### ISSUE THREAD

**Issue by Sheeri** [WildAid/o-fish-android#423]
in main/res/layout/fragment_vessel.xml the attachment icon is listed, but it needs a label for screen readers (right now if you click the attachment icon a screen reader says "unlabeled button").

I think this is the code fragment to add to:

                    <ImageView
                        android:id="@+id/btn_vessel_add_attachment"
                        android:layout_width="@dimen/add_attachment_icon_size"
                        android:layout_height="@dimen/add_attachment_icon_size"
                        android:src="@drawable/ic_add_attachment"
                        android:tint="@color/dark_grey"
                        app:layout_constraintEnd_toEndOf="@id/vessel_information_title"
                        app:layout_constraintTop_toTopOf="@id/vessel_information_title" />

And it should appear on the Vessel page for new/edit boardings.

**Comment by spacemonkey1101** [comment]
can i take this issue?

**Comment by Sheeri** [comment]
Hello @spacemonkey1101, thanks for offering to work on this! I have assigned it to you.

**Comment by subbramanil** [comment]
@Sheeri, Just wondering if I can help on this, assuming this is not fixed already.

**Comment by Sheeri** [comment]
Hi @subbramanil yes, you can help on it! I've assigned it to you.

**Comment by Sheeri** [comment]
@subbramanil how's it going? Do you need any help? we can set up office hours if you have some questions.

**Comment by subbramanil** [comment]
Hi @Sheeri, My apologies for the days without an update. Things happen in life, and I couldn't start working on the issue as I initially planned. It took a while to sort them out. 

This issue is a quick fix, and I noticed 35 other places where we could apply a similar fix. Let me know If I need to create a separate issue for each or maybe one issue where I can list down all the places, and We can figure out how we can address them.

<img width="361" alt="Screen Shot 2021-03-10 at 9 59 53 PM" src="https://user-images.githubusercontent.com/9157911/110733717-10db3b00-81ec-11eb-9c80-3f49b96b3d68.png">

**Comment by Sheeri** [comment]
Hi @subbramanil I will build and test this tomorrow. 

If you can list the places in this ticket, that would be great. (is this part of Android studio?)

**Comment by subbramanil** [comment]
You can get this list of places by going to the menu `Analyze -> Run Inspection by Name` and Type `Image without contentDescription.`

| Layout File                    | Image View ID                    | Image View Resource       |
| ------------------------------ | -------------------------------- | ------------------------- |
| activity_splash.xml            | -                                | ic_splash_logo            |
| fragment_activities.xml        | activity_activity_add_attachment | ic_add_attachment         |
| fragment_activities.xml        | activity_fishery_add_attachment  | ic_add_attachment         |
| fragment_activities.xml        | activity_gear_add_attachment     | ic_add_attachment         |
| fragment_add_crew.xml          | crew_member_action_icon          | ic_menu_edit              |
| fragment_basic_information.xml | -                                | ic_near_me                |
| fragment_full_image.xml        | full_image                       | -                         |
| fragment_home.xml              | image_search                     | ic_search_home            |
| fragment_home.xml              | image_user_status                | selector_status_indicator |
| fragment_home.xml              | image_seaerch                    | ic_search_home            |
| fragment_home.xml              | imageView                        | ic_draft_boarding         |
| fragment_home.xml              | image_amount_drafts              | shape_draft_amount        |
| fragment_login.xml             | login_logo                       | ic_login_logo             |
| fragment_risk.xml              | risk_action_icon                 | ic_add_attachment         |
| fragment_search.xml            | empty_result_image               | -                         |
| fragment_user_profile.xml      | image_user_status                | selector_status_indicator |
| fragment_vessel.xml            | btn_vessel_info_edit             | ic_edit                   |
| fragment_vessel.xml            | btn_vessel_delivery_edit         | ic_edit                   |
| fragment_vessel.xml            | btn_vessel_delivery_attachment   | ic_add_attachment         |
| fragment_violation.xml         | violation_seizures_attach        | ic_add_attachment         |
| item_crew_member.xml           | crew_member_edit_icon            | ic_edit                   |
| item_crew_member.xml           | crew_member_attachment_icon      | ic_add_attachment         |
| item_edit_catch.xml            | catch_action_edit                | ic_edit                   |
| item_edit_catch.xml            | catch_action_add_Attachment      | ic_add_attachment         |
| item_edit_report_note.xml      | note_action_add_attachment       | ic_add_photo_attachment   |
| item_edit_report_note.xml      | note_action_edit                 | ic_edit                   |
| item_edit_vessel_ems.xml       | btn_vessel_info_edit             | ic_edit                   |
| item_edit_vessel_ems.xml       | btn_vessel_info_attachment       | ic_add_attachment         |
| item_edit_violation.xml        | violation_action_edit            | ic_edit                   |
| item_edit_violation.xml        | violation_action_add_attachment  | ic_add_attachment         |
| item_photo.xml                 | item_edit_photo                  | -                         |
| item_photo.xml                 | item_edit_photo_remove           | ic_remove                 |
| item_record.xml                | record_vessel_draft_mark         | -                         |
| item_record.xml                | record_vessel_image              | ic_vessel_placeholder_2   |
| item_vessel_image.xml          | vessel_image                     | ic_vessel_profile         |


I added this list to a [google sheet](https://docs.google.com/spreadsheets/d/1DE3CY2t5ASIzfP7PSk09MusAuYvjpYzhrCCDHUImG0U/edit?usp=sharing). If you can provide the text to be added in 'English/French/Ukrainian, I can update them in proper places. Or I can use Google Translate to translate to other languages if the texts were given in one language.

**Comment by subbramanil** [comment]
I went over the list of places to identify where it's relevant to add a content description for the screen reader to pick up. I updated the google sheet with the findings and also possible text for the image views. Most of them are related to 'Edit' & 'Add attachments'.
